build/pkgs/prompt_toolkit/distros/conda.txt

```diff
@@ -1 +1 @@
-prompt_toolkit
+prompt_toolkit >=3.0.38
```

Comment:
Does conda let you put a space before the version? I don't see a space in any other `conda.txt`. I would say just delete it so we don't have to be bothered to check.

Reply:
No, the space must be removed

Reply:
I copied install-required.txt. oops ...